README.rst

```diff
@@ -62,7 +62,7 @@ Add following entry to ``west.yml`` file in ``manifest/projects`` subtree:
     - name: golioth
       path: modules/lib/golioth
       revision: main
-      url: git@github.com:golioth/zephyr.git
+      url: git@github.com:golioth/zephyr-sdk.git
```

Comment:
There is another link to update in section "Adding Golioth SDK for Zephyr to existing west project".

Reply:
Thanks for catching this. I came across this discrepancy while installing Golioth SDK this weekend and also mentioned it to @beriberikix. :)